### DISCUSSION TITLE
GLFW Error: EGL: Library not found    ERROR: glfwCreateWindow failed    window platform

### DISCUSSION THREAD

**Discussion by Jiajia-TaoTao** [kcleal/gw#65] [Q&A] [answered]
I am sorry,I meet some problem.I use GW at window11 platform and install it through MSYS。when I installed it, I don't see the "settings" buttom and it's command line。when I use GW,I input number about your own ref genome, There will be a crash on the interface。when I add some my ref genome in the ".gw.ini" file (like this:[genome] v6=E:\work\software_v2\another_plot4\mapping\DM_mapping\V6_chr.fa.gz   the “*.gz” file use bgzip compress), I meet a problem,like this:  GLFW Error: EGL: Library not found    ERROR: glfwCreateWindow failed .  please, could you help me with this?  thank you

**Comment by kcleal** [comment, marked as answer]
Hi @Jiajia-TaoTao,

The GLFW error is caused by a missing graphics library called EGL (which is part of openGL), for some reason it is not being found on your system when you start GW. It's possible you have the wrong environment set for msys2, https://www.msys2.org/docs/environments/. I don't use msys2 much so I don't know how it interacts with openGL.
